platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/CriticalQuorumTest.java

```diff
@@ -78,8 +81,17 @@ public String toString() {
      * Build an event containing just the data required for this test.
      */
     private static EventImpl buildSimpleEvent(@NonNull final NodeId creatorId, final long roundCreated) {
+
+        final EventDescriptor selfDescriptor = new EventDescriptor(new Hash(), creatorId, 0, -1);
+        final EventDescriptor otherDescriptor = new EventDescriptor(new Hash(), new NodeId(0), 0, -1);
         final BaseEventHashedData baseEventHashedData = new BaseEventHashedData(
-                new BasicSoftwareVersion(1), creatorId, 0, 0, (byte[]) null, null, Instant.now(), null);
+                new BasicSoftwareVersion(1),
+                creatorId,
+                selfDescriptor,
+                Collections.singletonList(otherDescriptor),
+                -1,
+                Instant.now(),
+                null);
```

Comment:
you can use `GossipEventBuilder` for this

Reply:
I'd recommend you don't spend too much time on this test. Critical quorum is disabled in `0.44`, and I intend to delete it in `0.45`.